package.json

```
@@ -23,7 +23,7 @@
         }
     },
     "engines": {
-        "node": ">=10 <13"
+        "node": ">=16"
```

change wasn't needed

```suggestion
        "node": ">=10 <13"
```